pom.xml

```diff
@@ -635,6 +636,7 @@
       <plugin>
         <groupId>org.projectlombok</groupId>
         <artifactId>lombok-maven-plugin</artifactId>
+        <version>${lombokPluginVersion}</version>
```

Comment:
Unit Test `testInlineProperty` uses this pom.xml file to run the tests, that's why I created the `lombokPluginVersion` property.